server/http.go

```diff
@@ -384,29 +402,33 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 
 	meetingID := zoomUser.Pmi
 
-	createdPost, appErr := p.postMeeting(user, meetingID, req.ChannelID, req.Topic)
-	if appErr != nil {
-		http.Error(w, appErr.Error(), appErr.StatusCode)
-		return
-	}
-
-	if appErr = p.API.KVSet(fmt.Sprintf("%v%v", postMeetingKey, meetingID), []byte(createdPost.Id)); appErr != nil {
-		http.Error(w, appErr.Error(), appErr.StatusCode)
+	err = p.postMeeting(user, meetingID, req.ChannelID, req.Topic)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
-	meetingURL := p.getMeetingURL(meetingID)
+	meetingURL := p.getMeetingURL(meetingID, userID)
 
 	_, err = w.Write([]byte(fmt.Sprintf(`{"meeting_url": "%s"}`, meetingURL)))
 	if err != nil {
 		p.API.LogWarn("failed to write response", "error", err.Error())
 	}
 }
 
-func (p *Plugin) getMeetingURL(meetingID int) string {
-	meeting, err := p.zoomClient.GetMeeting(meetingID)
-	if err == nil {
-		return meeting.JoinURL
+func (p *Plugin) getMeetingURL(meetingID int, userID string) string {
+	if p.configuration.EnableLegacyAuth {
+		meeting, err := p.zoomClient.GetMeeting(meetingID)
+		if err == nil {
+			return meeting.JoinURL
+		}
```

Comment:
I'm a bit worried that an possible error just get's omitted. Will using `config.ZoomURL` result in a correct URL?

Is there a better way to handle this? e.g. logging or returning?

Reply:
This is done mainly to add the password to the meeting. Using config.ZoomURL should always work, but will not show the password.

Reply:
Got it. Does it make sense to log the error then to help with debugging?